src/R4MvcHostApp/Areas/Admin/Controllers/UsersController.cs

```diff
@@ -3,15 +3,37 @@
 using System.Linq;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Rendering;
 
 namespace R4MvcHostApp.Areas.Admin.Controllers
 {
     [AdminArea]
     public partial class UsersController : Controller
     {
+
+        static UsersController()
+        {
+            ModelUnbinderHelpers.ModelUnbinders.Add(typeof(Index2ViewModel), new R4Mvc.ModelUnbinders.SimplePropertyModelUnbinder());
```

Comment:
Even though it's a test project, this is by far not the best place to add this registration =/

Since this is configuration data, this should be in Startup.cs, but there are also other issues with putting it there (tbd)

Reply:
Yes in production we usually register this on startup/global, i was just isolating use cases.